src/views/Accessibility/AccessibiltyRequest/Create/index.tsx

```diff
@@ -124,19 +131,18 @@ const Create = () => {
                             id="508Request-IntakeId"
                             options={projectComboBoxOptions}
                             onChange={(intakeId: any) => {
-                              const selectedSystem = systems.find(
-                                system => system.node.id === intakeId
-                              );
-                              setFieldValue('intakeId', intakeId || '');
-                              setFieldValue(
-                                'businessOwner.name',
-                                selectedSystem?.node.businessOwner.name || ''
-                              );
-                              setFieldValue(
-                                'businessOwner.component',
-                                selectedSystem?.node.businessOwner.component ||
-                                  ''
-                              );
+                              const selectedSystem = systems[intakeId];
```

Comment:
Nice! It's a small decrease in runtime, but it saves us from looping on every choice.